src/main/scala/org/scalajs/tools/tsimporter/Importer.scala

```diff
@@ -245,6 +245,10 @@ class Importer(val output: java.io.PrintWriter) {
         }
         TypeRef(baseTypeRef, targs map typeToScala)
 
+      case ObjectType(List(IndexMember(_, TypeRefTree(CoreType("string"),_), valueType))) =>
+        val valueTpe = typeToScala(valueType)
+        TypeRef(QualifiedName.Dictionary,List(valueTpe))
```

Comment:
Please add one space after `,` (there is one missing in the first line, and one in the third line)